### PR TITLE
FUSETOOLS2-2299: Display variables values during debugging for Exchange scope

### PR DESCRIPTION
**Pull request by djelinek**
Camel Variables
- https://camel.apache.org/manual/variables.html
- https://camel.apache.org/blog/2024/02/camel44-whatsnew/#variables
- https://raymondmeester.medium.com/camel-variables-new-ways-to-manipulate-data-pipelines-2bf3d31cb26c

-----

This is how the Event Message looks like
```xml
<backlogTracerEventMessage>
  <uid>1</uid>
  <first>false</first>
  <last>false</last>
  <rest>false</rest>
  <template>false</template>
  <timestamp>2024-03-14T15:32:03.349+0100</timestamp>
  <elapsed>928</elapsed>
  <threadName>Camel (camel-1) thread #6 - timer://foo</threadName>
  <done>false</done>
  <failed>false</failed>
  <location>myapp.java:12</location>
  <routeId>route1</routeId>
  <toNode>log1</toNode>
  <exchangeId>6A28393E31D32D8-0000000000000000</exchangeId>
  <message exchangeId="6A28393E31D32D8-0000000000000000" exchangePattern="InOnly" exchangeType="org.apache.camel.support.DefaultExchange" messageType="org.apache.camel.support.DefaultMessage">
    <exchangeVariables>
      <exchangeVariable key="item" type="java.lang.String">world</exchangeVariable>
      <exchangeVariable key="name" type="java.lang.String">Dominik</exchangeVariable>
    </exchangeVariables>
    <body>[Body is null]</body>
  </message>
</backlogTracerEventMessage>
```

This is how the variables for the Exchange scope are displayed
Screenshot 2024-03-14 at 15 39 12

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ To test with the Client integration of Eclipse IDE:
   }
   ```
   - Click `Debug`
+- Connect Remote Java Application to debug Camel DAP server project
+  - Create a `Remote Java Application` configuration
+  - In `project` filed, set `camel-dap-server`
+  - In `port` filed, set `3000`
```
Comment (apupier) [inline Contributing.md:38]: filed --> field

File: Contributing.md [hunk 2]
```diff
@@ -32,6 +32,12 @@ To test with the Client integration of Eclipse IDE:
...
+  - Select `Allow termination of remote VM`
+  - Click `Debug`
```
Comment (apupier) [inline Contributing.md:40]: can be provided in a different PR

Comment (djelinek) [reply Contributing.md:40]: ok 👍🏻 I will provide in another, no problem

File: src/test/resources/basic-withroutes.xml [hunk 3]
```diff
@@ -16,6 +16,12 @@
 		<setProperty name="property2">
 			<constant>value of property 2</constant>
 		</setProperty>
+		<setVariable name="var1">
+			<constant>value of variable 1</constant>
+	    </setVariable>
+	    <setVariable name="var2">
+	        <constant>value of variable 2</constant>
+	    </setVariable>
```
Comment (apupier) [inline src/test/resources/basic-withroutes.xml:24]: I there are some spaces mixed with the tabs

**Review by apupier — APPROVED** [review]

File: src/test/resources/basic-withroutes.xml (hunk 3 above)
Comment (djelinek) [reply src/test/resources/basic-withroutes.xml]: nice catch --> fixed
